### ISSUE TITLE
Windows binary

### ISSUE THREAD

**Issue by rodber** [xrdebug/xrdebug#30]
Feel free to use this issue to let me know if you are interested in a Windows binary package.

**Comment by w33zy** [comment]
Yes, I would be interested in a Windows binary package. WordPress developers would also love this.

**Comment by eness** [comment]
I also do interest. Thanks for such a great lib!

**Comment by rodber** [comment]
@w33zy @eness 

From [upstream](https://github.com/dixyes/lwmbs/actions/runs/7578726225) it looks that I could build a Windows binary. Of course I need to give it a try but it seems that soon we could get native Windows support for xrDebug!

**Comment by rodber** [comment]
Hey @eness and @w33zy 

Have you tried running this via WSL?

To run via WSL simply download the Linux binary for your CPU arch and run this on CMD:

```sh
wsl ./xrdebug <options>
```

I ask because the Windows binary will complain to run this and will panic all over Windows Defender which is far from what end users are expecting.

I tried yesterday and apart from configuring WSL the process was quite easy and now I can run any Linux binary on Windows.

**Comment by rodber** [comment]
I just wanted to follow up on the request for a Windows build of xrdebug. @eness @w33zy 

Initially, I thought, ‘Sure, how hard can it be?’ Turns out, extremely hard. After jumping through endless hoops to compile it for multiple OSs, I finally got it working… only to have Windows Defender enthusiastically label it as malware but despite that, the actual issue is that such way of compiling the software is too much hassle and breaks too often.

So, naturally, I did what any reasonable person would do: I learned an entirely new programming language, rewrote the entire project from scratch, and built a native Windows binary. It's been quite a journey, but now I can distribute native binaries for all platforms!

[xrdebug_x86.exe.zip](https://github.com/user-attachments/files/18220183/xrdebug_x86.exe.zip)

Let me know how it goes.
